code/modules/clothing/under/ties.dm

```diff
@@ -391,8 +391,44 @@
 		hold = null
 	. = ..()
 
-/obj/item/clothing/tie/storage/attack_hand(mob/user as mob)
-	if (has_suit)	//if we are part of a suit
+/obj/item/clothing/tie/storage/on_attached(obj/item/clothing/under/S, mob/user)
+	. = ..()
+	has_suit.verbs += /obj/item/clothing/tie/storage/verb/toggle_draw_mode
+
+/obj/item/clothing/tie/storage/on_removed()
+	has_suit.verbs -= /obj/item/clothing/tie/storage/verb/toggle_draw_mode
+	return ..()
+
+/obj/item/clothing/tie/storage/verb/toggle_draw_mode()
+	set name = "Switch Storage Drawing Method"
+	set category = "Object"
+	set src in usr
+	if(!isliving(usr))
+		return
+	if(usr.stat)
+		return
+
+	var/obj/item/clothing/tie/storage/H = src
```

Comment:
The start of this holster mirroring was fine, but the followup not so much.
Keep, `on_attached()` and `on_removed()`, rename this verb to something like `toggle_webbing_draw()` or something more clearly distinct than the already-existing same-name verb, and instead of handling it here, call the other verb here. For example:
```
/obj/item/clothing/tie/storage/verb/toggle_webbing_draw()
	set name = "Switch Webbing Drawing Method"
	set category = "Object"
	hold?.toggle_draw_mode()
```
Might require the hold to get a name if it doesn't have one already, in its creation.